examples/js/exporters/GLTFExporter.js

```diff
@@ -331,21 +332,50 @@ THREE.GLTFExporter.prototype = {
 		 * Serializes a userData.
 		 *
 		 * @param {THREE.Object3D|THREE.Material} object
-		 * @returns {Object}
+		 * @param {Object} gltfProperty
 		 */
-		function serializeUserData( object ) {
+		function serializeUserData( object, gltfProperty ) {
+
+			if ( ! object.userData || Object.keys( object.userData ).length === 0 ) {
```

Comment:
Minor thing. Why are we checking `object.userData`? Shouldn't `object.userData` always be there?

https://github.com/mrdoob/three.js/blob/dev/src/core/Object3D.js#L99

Reply:
The object may also be a `geometry`, and I think A-Frame needed to use a new version of GLTFExporter with an older version of three.js for a while. The most recent A-Frame release is on r101, so perhaps it isn't necessary now.

Reply:
So is the check for `object.userData` still necessary or can I remove it?

Reply:
It can be removed. 👍